tilt_modules/cert_manager/Tiltfile

```diff
@@ -26,27 +26,26 @@ spec:
 """
 
 # Deploys cert manager to your environment
-def deploy_cert_manager(registry="quay.io/jetstack", version="v0.16.1", load_to_kind=False, kind_cluster_name="kind"):
+def deploy_cert_manager(registry="quay.io/jetstack", version="v1.1.0", load_to_kind=False, kind_cluster_name="kind"):
     silent=True
+    if version.startswith('v0'):
```

Comment:
This is kind of weak, but I don't have a better alternative in mind.
I will try to do some research on stark

Reply:
I know, I was thinking about that. Probably the best solution on Python is to use the semver package, but this is not a default language package and I do not know if this works on Tilt.
Maybe I can improve the check with a regex...